modules/installation-special-config-storage.adoc

```diff
@@ -196,7 +197,7 @@ openshift:
 ----
 +
 <1> For control plane configurations, replace `worker` with `master` in both of these locations.
-<2> On ppc64le nodes, set this field to `ppc64le`.  On all other nodes, this field can be omitted.
+<2> On ppc64le nodes, set this field to `ppc64le`. On arm64 based nodes, set this field to `aarch64`. On all other nodes, this field can be omitted.
```

Comment:
The same comment as above. I suppose this is an old statement when only ppc64le and x86_64 were available.